stream_alert/alert_processor/outputs.py

```diff
@@ -803,3 +805,296 @@ def dispatch(self, **kwargs):
                                  Payload=alert_string)
 
         return self._log_status(resp)
+
+@output
+class JiraOutput(StreamOutputBase):
+    """JiraOutput handles all alert dispatching for Jira"""
+    __service__ = 'jira'
+
+    DEFAULT_HEADERS = {'Content-Type': 'application/json'}
+    LOGIN_ENDPOINT = 'rest/auth/1/session'
+    SEARCH_ENDPOINT = 'rest/api/2/search'
+    ISSUE_ENDPOINT = 'rest/api/2/issue'
+    COMMENT_ENDPOINT = 'rest/api/2/issue/{}/comment'
+
+    def __init__(self, *args, **kwargs):
+        StreamOutputBase.__init__(self, *args, **kwargs)
+        self.base_url = None
+        self.auth_cookie = None
```

Comment:
Maybe make these instance properties protected? ie - `self._ auth_cookie`, since they should never be accessed from outside an instance of this class.